apps/prairielearn/src/ee/pages/linkedLtiUserRequired/linkedLtiUserRequired.html.ts

```diff
@@ -0,0 +1,64 @@
+import { html } from '@prairielearn/html';
+import { run } from '@prairielearn/run';
+
+import { PageLayout } from '../../../components/PageLayout.html.js';
+import type { Lti13Instance } from '../../../lib/db-types.js';
+
+export function AuthzRequireLinkedLtiUser({
+  instancesWithMissingIdentities,
+  resLocals,
+}: {
+  instancesWithMissingIdentities: { lti13_instance: Lti13Instance; lti13_user_id: string | null }[];
+  resLocals: Record<string, any>;
+}) {
+  return PageLayout({
+    pageTitle: 'Missing LTI connection',
+    navContext: {
+      type: 'student',
+      page: undefined,
+    },
+    resLocals,
+    content: html`
+      <div class="card mb-4">
+        <div class="card-header bg-warning">
+          <h1>Missing LTI connection</h1>
+        </div>
+        <div class="card-body">
+          ${run(() => {
+            if (instancesWithMissingIdentities.length === 1) {
+              const instance = instancesWithMissingIdentities[0].lti13_instance;
+              return html`
+                <p>
+                  To continue, please log into PrairieLearn via <strong>${instance.name}</strong>.
+                  This creates a connection between your PrairieLearn account and your
+                  ${instance.name} account.
```

Comment:
I talked to Firas on Slack about this, he recommended changing the second sentence to this:

> Typically, the PrairieLearn link will be available from without your course in $NAME once you are officially registered.

We can apply the same idea to the 2+ case below if we like this.